### PR TITLE
add vertical diffusion for momentum

### LINKED ISSUE #469: Add surface boundary condition on momentum
Currently, there is no boundary condition on momentum. We will start by adding a drag with a constant coefficient, similar as the one for energy and tracers. This is required for the idealized moist aquaplanet.

### PR DESCRIPTION
**Pull request by szy21**
Closes #469

### CODE REVIEW

File: examples/hybrid/sphere/baroclinic_wave_utilities.jl [hunk 1]
```diff
@@ -419,6 +442,35 @@ function vertical_diffusion_boundary_layer_tendency!(Yₜ, Y, p, t)
             params,
         )
 
+    # Surface Fluxes: 
+    # Define a local `state` (e.g. SurfaceFluxes.jl src.jl l281)
+    # SC = SurfaceFluxes.surface_conditions(param_set, sc::AbstractSurfaceConditions, uft::UF.Businger(), scheme=FVScheme())
+    # then, get vars from SC.<varname> (Surface Fluxes is pointwise so broadcasts should work internally)
+    # Currently, possible output varnames are L_MO, shf, lhf, buoy_flux, ρτxz, ρτyz, ustar, Cd, Ch
```
[the remark this answers is not in the document]
Comment (szy21) [reply examples/hybrid/sphere/baroclinic_wave_utilities.jl:449]: I have a slight preference of removing this if you don't mind.

**Comment by szy21** [comment]
> Looks fine to me, but can we wait to merge this after #433?

I would like to merge this soon, but the ci is broken anyway. Does this conflict with 433?

**Comment by charleskawczynski** [comment]
> I would like to merge this soon, but the ci is broken anyway. Does this conflict with 433?

Actually I don't there there will be many conflicts

**Pull request merged**